commonknowledge/meep-intelligence-hub
# feat: make import all, update all and map reports work for large datasets

## Description
- Make the import and refresh all jobs spawn child jobs
- Split up the front-end map query for getting all the points into an ID query and a properties query

## How Can It Be Tested?
Connect the 50k airtable I have prepared to MEEP, click Import All, and then Refresh All.
Create a report and check the map can be used.
